sources/install/package_mobile.sh

```diff
@@ -35,48 +53,82 @@ function install_smali(){
     add-to-list "smali,https://github.com/JesusFreke/smali,A tool to disassemble and assemble Android's dex files"
 }
 
-function install_dex2jar(){
+function install_dex2jar() {
+    # CODE-CHECK-WHITELIST=add-aliases
     colorecho "Installing dex2jar"
-    wget https://github.com/pxb1988/dex2jar/releases/latest/download/dex2jar-2.1.zip -O /tmp/dex2jar.zip
+    wget https://github.com/pxb1988/dex2jar/releases/download/v2.3/dex2jar-v2.zip -O /tmp/dex2jar.zip
```

Comment:
latest is: https://github.com/pxb1988/dex2jar/releases/download/v2.4/dex-tools-v2.4.zip

Reply:
Updating